.github/workflows/test-full-sync.yml

```
@@ -142,6 +140,11 @@ jobs:
         with:
           short-length: 7
 
+      - name: Format network string
+        run: |
+          NETWORK_CAPS=${{ github.event.inputs.network || env.NETWORK }}
+          echo "NETWORK=${NETWORK_CAPS,,}" >> $GITHUB_ENV
+
```

```suggestion
      - name: Downcase network name for disks
        run: |
          echo LOWER_NET_NAME="${{ github.event.inputs.network || env.NETWORK }}" | awk '{print tolower($0)}' >> $GITHUB_ENV
```

I prefer the way you're doing this. I'll even create a PR to substitute the other implementations available in other YAMLs, as the one available here https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/test.yml#L368-L370

But if we do it here, we'll have different ways to do it in a single point in time, making it harder for global substitution.